acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/httptransaction/HttpRequestFactory.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
nit: package name should be httpTransaction

Reply:
or maybe just http?

Reply:
I looked through the codebase and couldn't find examples of mixed case package names, found the opposite with all lower case. The transaction part is being copied from tech.pegasys.pantheon.tests.acceptance.dsl.transaction as this is a different form of the same thing.

Reply:
👍